src/components/VenueDetail/index.js

```diff
@@ -0,0 +1,133 @@
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+import useSWR from "swr";
+import { useFilterStore } from "../../store";
+import { PlaceBig, TheaterBig } from "../../utils/icons";
+import BackLink from "../BackLink";
+import DistanceWidget from "../DistanceWidget";
+import EventList from "../EventList";
+import StyledHeader from "../StyledHeader";
+import StyledHeadline from "../StyledHeadline";
+import StyledIconLink from "../StyledIconLink";
+import StyledWidgetContainer from "../StyledWidgetContainer";
+
+const StyledSection = styled.section`
+  padding: 0.5rem;
+  margin: 0.1rem;
+  border: 2px solid black;
+  width: 98%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(4, 1fr);
+  gap: 0.5rem;
+`;
+
+const StyledAddressSection = styled.div`
+  grid-column: 1 / 4;
+  grid-row: 3;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+`;
+
+const StyledDescription = styled.section`
+  padding: 0.5rem;
+  margin: 0.1rem;
+  border: 2px solid black;
+  width: 98%;
+`;
+
+const StyledFrame = styled.div`
+  grid-column: span 4;
+  grid-row: 1 / 3;
+  margin-left: 1rem;
+`;
+
+const StyledImage = styled(Image)`
+  border-radius: 12px;
+  box-shadow: 0 1px 1px hsl(0deg 0% 0% / 0.075),
+    0 2px 2px hsl(0deg 0% 0% / 0.075), 0 4px 4px hsl(0deg 0% 0% / 0.075),
+    0 8px 8px hsl(0deg 0% 0% / 0.075), 0 16px 16px hsl(0deg 0% 0% / 0.075);
+`;
+
+export default function VenueDetail({
+  venue,
+  range,
+  distance,
+  currentLocation,
+}) {
+  const eventSort = useFilterStore((state) => state.eventSort);
+  const { data } = useSWR(
+    `/api/events/events?venueId=${venue.id}&sort=${eventSort}&locale=*&countryCode=DE`
+  );
+  const events = data?._embedded.events;
+  const venueImage = venue.images
+    ? venue.images.find((image) => image.ratio === "16_9")
+    : null;
+  const venueImageUrl = venueImage ? venueImage.url : null;
+
+  return (
+    <>
+      <StyledHeader>
+        <BackLink />
+        <StyledHeadline aria-label={venue.name}>{venue.name}</StyledHeadline>
+        <StyledIconLink href="/location" aria-label="go to locations list">
+          <PlaceBig />
+        </StyledIconLink>
+      </StyledHeader>
+      <StyledSection style={{ background: "#f0f0f0" }}>
+        {venueImage ? (
+          <StyledFrame>
+            <StyledImage
+              src={venueImageUrl}
+              alt={venue.name}
+              width={250}
+              height={250 / (16 / 9)}
+            />
+          </StyledFrame>
+        ) : (
+          <TheaterBig />
+        )}
+        <StyledAddressSection>
+          <address aria-label="address">
+            {venue.address.line1}, {venue.city.name}
+          </address>
+        </StyledAddressSection>
+        <StyledWidgetContainer
+          style={{
+            gridColumn: "span 4",
+            gridRow: 4,
+          }}
```

Comment:
Is there a reason you define these styles inline, instead of in your styled component?

Reply:
just some pseudo semantic bs tho :p

Reply:
I'm not sure about the styling yet. This is experimental and will be refactored in a styling update. 💅